research/build-reverse-dependencies/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Build reverse dependencies in Packit
+authors: mmassari
+---
```

Comment:
General comment, what are the project structures that would support such reverse dependencies. Firstly w.r.t. where the builds will live
- copr project with multiple packages
  - Easy to pick up the latest changes
  - Need to manually clean the project build, otherwise deleted packages will still be used (e.g. a package is already downstream and the old copr package is no longer needed)
  - Bootstrap packages would be weird to handle
- multiple copr projects linked via `additional_repos`
  - Maximum control of the packages
  - Messy to navigate
- koji based builds
  - Good control of packages to be used
  -  Not universal for upstream projects

Reply:
Re: side-tag, that's for the case of making the reverse dependency build a production workflow right? Can scratch builds also have side-tag or equivalent?

Reply:
> Can scratch builds also have side-tag or equivalent?
You can assign a side tag to a scratch build.

If for production workflow you mean you need to have your commit released in dist-git. This is already supported in Koshei. We are looking for a way to have the workflow triggered before a new release in dist-git.

Reply:
I'm thinking w.r.t. the end result. Will it eventually be used in a bodhi update, or will it be used only for testing dependencies. The former might be needed only sporadically, like for SOVERSION bump, etc. but the latter could be used more freely. Also the former could be covered by the dist-git label workflow